### PR TITLE
Updating text guide

### PR DESCRIPTION
**Pull request by JoogabYun**
### Change Description ###

I changed the sample code.
Added description of default value.

### CODE REVIEW

File: docs/application/dotnet/guides/nui/text.md [hunk 1]
```diff
@@ -207,20 +207,20 @@ To use the text-decoration, set the applicable property:
...
-    shadow.Add("color", new PropertyValue("black"));
+    shadow.Add("offset", new PropertyValue(new Vector2(2, 2));
+    shadow.Add("color", new PropertyValue(Color.Black));
     label2.Shadow = shadow;
 
     label3.Text = "Text with Color Shadow";
     PropertyMap shadow = new PropertyMap();
-    shadow.Add("offset", new PropertyValue("1 1"));
-    shadow.Add("color", new PropertyValue("red"));
+    shadow.Add("offset", new PropertyValue(new Vector2(1, 1));
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/text.md:222]: Would you please check the parenthesis?
```suggestion
    shadow.Add("offset", new PropertyValue(new Vector2(1, 1)));

```

File: docs/application/dotnet/guides/nui/text.md [hunk 2]
```diff
@@ -207,20 +207,20 @@ To use the text-decoration, set the applicable property:
...
-    shadow.Add("color", new PropertyValue("black"));
+    shadow.Add("offset", new PropertyValue(new Vector2(1, 1));
+    shadow.Add("color", new PropertyValue(Color.Black));
     label1.Shadow = shadow;
 
     label2.Text = "Text with Bigger Shadow";
     PropertyMap shadow = new PropertyMap();
-    shadow.Add("offset", new PropertyValue("2 2"));
-    shadow.Add("color", new PropertyValue("black"));
+    shadow.Add("offset", new PropertyValue(new Vector2(2, 2));
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/text.md:216]: Would you please check the parenthesis?
```suggestion
    shadow.Add("offset", new PropertyValue(new Vector2(2, 2)));

```

File: docs/application/dotnet/guides/nui/text.md [hunk 3]
```diff
@@ -207,20 +207,20 @@ To use the text-decoration, set the applicable property:
 
     label1.Text = "Text with Shadow";
     PropertyMap shadow = new PropertyMap();
-    shadow.Add("offset", new PropertyValue("1 1"));
-    shadow.Add("color", new PropertyValue("black"));
+    shadow.Add("offset", new PropertyValue(new Vector2(1, 1));
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/text.md:210]: Would you please check the parenthesis?
```suggestion
    shadow.Add("offset", new PropertyValue(new Vector2(1, 1)));

```

File: docs/application/dotnet/guides/nui/text.md [hunk 4]
```diff
@@ -244,7 +244,7 @@ To use the text-decoration, set the applicable property:
     label1.Underline = textStyle;
     ```
 
-    By default, the underline height is based on the font metrics. For example, the following text figures are in one pixel height:
+    By default, the underline height is based on the font metrics and the minimum height is one pixel. And the underline color is based on the text color. For example, the following text figures are in one pixel height and the color is the same as the text color:
```
Comment (surbhig12) [inline docs/application/dotnet/guides/nui/text.md:247]: Suggest rephrasing to:

By default, the underline height is based on the font metrics and the minimum height is one pixel. The underline color is based on the text color. For example, the following text figures are in one pixel height and the color is the same as the text color:

Please verify

Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/text.md:247]: @surbhig12 Thanks for rewriting.

@JoogabYun Would you please check @surbhig12's comment and apply it?

**Comment by mijin85cho** [comment]
Thanks for the update, @JoogabYun .
@surbhig12 , as your comment is applied, would you please check again with the latest?
If you see no more issue, please approve this PR to process the next step.

**Pull request merged**